ui/assets/tad.css

```diff
@@ -0,0 +1,33 @@
+.tad_hidden {
```

Comment:
For now CSS is fine, but i would personally go for Sass for more complex files if we need that in the future

Reply:
Yes, bear with us on this.. we "grabbed" a rendered version of the Algoritmeregister in the most minimal version we could, which still was a whole lot. The pages are generated by a backend, so we basically "hacked ourself in".. if and when this takes on others forms, we can and will think about these kind of questions.

Reply:
A, then it makes more sense. Thanks for explaining. I will use this information in the rest of my review :D